R/Rules-methods.R

```diff
@@ -1033,6 +1033,81 @@ setGeneric("maxDose",
 )
 
 
+## --------------------------------------------------
+## The maximum allowable absolute increments in intervals method
+## --------------------------------------------------
+
+##' @describeIn maxDose Determine the maximum possible next dose based on
+##' absolute increments
+##'
+##' @example examples/Rules-method-maxDose-IncrementsAbsolute.R
+setMethod(
+  "maxDose",
+  signature(increments = "IncrementsAbsolute", data = "Data"),
+  function(increments, data) {
+    if (length(data@x) == 0) {
+      # print(paste0("Empty grid: returning dose index ", increments@increments[1], ": ", data@doseGrid[increments@increments[1]]))
+      return(data@doseGrid[min(length(data@doseGrid), increments@increments[1])])
+    }
+    usedMax <- max(data@x)
```

Comment:
we use snake style rather than camel for naming variables, so it should be `used_max`.